### PR TITLE
Fixes #7070: osname should be called inventory os

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7070

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
This needs a rebase !

**Comment by peckpeck** [comment]
PR updated

File: scenario/base.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # Generic tests
 for host in scenario.nodes():
   hostinfo = scenario.platform.hosts[host].info
-  osname = hostinfo['osname'] if 'osname' in hostinfo else ""
+  osname = hostinfo[''] if 'inventory-os' in hostinfo else ""
```
Comment (VinceMacBuche) [inline scenario/base.py]: should be:  hostinfo['inventory-os']  ;)

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**